core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -866,6 +867,28 @@ def _compute_permutation_importance(self, X, y, features: list, eval_metric=None
             transform_func=transform_func, transform_func_kwargs=transform_func_kwargs, silent=silent, **kwargs
         )
 
+    # TODO: Pick up from here, rename to save, implement _get_compiler(), etc.
+    def compile(self, path: str = None, verbose=True) -> str:
+        if path is None:
+            path = self.path
+        file_path = path + self.model_file_name
+
+        save_in_pkl = True
+        if self.model is not None:
+            self._compiler = self._get_compiler()
+            if self._compiler is not None:
+                self.model = self._compiler.compile(obj=self, path=path)
+                save_in_pkl = self._compiler.save_in_pkl
+        _model = self.model
+        if not save_in_pkl:
+            self.model = None
+        save_pkl.save(path=file_path, object=self, verbose=verbose)
+        self.model = _model
+        return path
```

Comment:
This should be marked as 

```
# TODO: PoC, refactor prior to v0.6 release
#  Add Documentation
#  Define when this should be called
#  Is it called during fit? Is it called after fit?
#  Can it be called as a post-fit operation on an already fit predictor on a per-model basis?
#  Does this call overwrite the existing model or make a new one?
```

Reply:
Added these lines on top of the function.